guide/token-formats.md

```diff
@@ -0,0 +1,99 @@
+# Access Token Formats in OpenIddict
+
+OpenIddict can be configured to use three access token formats:
+
+ - opaque tokens (default)
+ - reference tokens
+ - JWTs (Json Web Tokens)
+ 
+ Tokens differ in what they look like and how they are validated. The default tokens will work fine in most use cases. There are times, however, where the other token formats would be preferred or required.
+ 
+ The goal of this guide:
+ 
+ - **Not** to explain each token format in detail
+ - Help users decide which formats to use in what use cases
+ - Show how to configure OpenIddict to produce them 
+ - Show how to configure api servers (resource servers) to validate and consume them
+ 
+ **Note: Identity tokens are always JWTs, according to spec.**
+ 
+ 
+ ## Opaque tokens (default)
+
+ The default access tokens are opaque tokens. They are encrypted and signed by the authorization server using the ASP.NET Core Data Protection stack. Their contents can only be inspected by the authorization server or another server sharing the same ASP.NET Core Data Protection configuration.
+
+ These are "proprietary" tokens that are not meant to be read or verified by a third-party, as the token format is not standard and necessarily relies on symmetric signing and encryption.
+
+We use this format for authorization codes and refresh tokens. They are only meant to be consumed by OpenIddict itself.
+
+ ### Use cases
+
+  - When there's no separate resource server
+  - When resource servers are .NET only and can share ASP.NET Core's DataProtection configuration with auth server
+
+  ### Benefits
+  - No additional configuration required
+  - Uses OpenIddict's built-in validation
+  - Resource servers can validate tokens without having to contact auth server
+  - Tokens are encrypted so no one can inspect the token, e.g. if tokens somehow end up in your logs somewhere or are intercepted somehow
+
+  ### Drawbacks
+  - Proprietary format, so if you add non .NET core resource servers in the future you need to swith to JWTs
```

Comment:
For local/direct validation, yes. But one could also opt for **token introspection** to ask the authorization server to validate on its behalf.